lib/src/input/figma/entities/layers/figma_frame.dart

```diff
@@ -104,13 +90,10 @@ class FigmaFrame extends FigmaChildrenNode
   @JsonKey(ignore: true)
   List points;
 
-  @JsonKey(name: 'fills')
-  List fillsList;
-
   @override
   FigmaNode createFigmaNode(Map<String, dynamic> json) {
     var node = FigmaFrame.fromJson(json);
-    node.style = StyleExtractor().getStyle(json);
+    // node.style = StyleExtractor().getStyle(json); TODO: Fix
```

Comment:
What needs to be fixed here? 🤔

Reply:
This got obsolete since now we get the style from the json